Libraries/D3D12RaytracingFallback/src/RayTracingHelper.hlsli

```diff
@@ -25,6 +25,9 @@ static const int IsLeafFlag = 0x80000000;
 static const int IsProceduralGeometryFlag = 0x40000000;
 static const int LeafFlags = IsLeafFlag | IsProceduralGeometryFlag;
 
+#define COMBINE_LEAF_NODES 1
+static const int IsCollapseChildren = 0x80000000; // for extracting HierarchyNode::bCollapseChildren
```

Comment:
If this is intended to be exclusive to some other value, it should be all defined closely together.